pkg/sync/v14/data/ClusterRole.rbac.authorization.k8s.io/customer-admin-cluster.yaml

```diff
@@ -171,20 +171,8 @@ rules:
   - metrics.k8s.io
   resources:
   - nodes
+  - pods
   verbs:
   - get
   - list
   - watch
-- apiGroups:
-  - autoscaling
-  resources:
-  - horizontalpodautoscalers
-  verbs:
-  - create
-  - delete
-  - deletecollection
-  - get
-  - list
-  - patch
-  - update
-  - watch
```

Comment:
@Makdaam can you PTAL?

Reply:
note, that this section I added earlier in this PR, it is not in diff against master: https://github.com/openshift/openshift-azure/pull/2099/files#diff-e024d6fd3fbeb1611a5f7f4178de9ae9